docs/diagrams/help/HelpSequenceDiagram.puml

```diff
@@ -0,0 +1,65 @@
+@startuml
+!include style.puml
```

Comment:
Perhaps this diagram should be included in the DG as well, as right now it is not used.